### DISCUSSION TITLE
epiWaves●≈ - Screens and Health: EMF Scentist Devra Davis Visits Norway

### DISCUSSION THREAD

**Discussion by giscus** [cortexR/blog-docs#3] [General]
# epiWaves●≈ - Screens and Health: EMF Scentist Devra Davis Visits Norway

https://madsrohde.com/posts/en/davis-disconnect-oslo/

**Comment by ghost** [comment]
"More curiosity needed"
As long ICNIRP, the International Commission on Non Ionizing Radiation Protection, has not been exposed to the public in the correct way, which means that their lies are exposed and their "science" does not exist, there will be no curiosity.
FCC is mentioned, but FCC has copied and pasted the ICNIRP "safety" guidelines, which are unscientific.
U n s c i e n t i f i c. 
This has been pointed out, analysed, by ICBE-EMF, https://icbe-emf.org/ , the International Commission on the Biological Effects of EMF. Video with 22 subtitles, available on YouTube: https://www.youtube.com/watch?v=9MicoRBN510
Collection of articles, papers and videos about ICNIRP, with evidence of fraud, lies, and their unscrupulous contributors: https://multerland.blog/2019/06/02/icnirp/

Telecom rules the world, 
Telecom rules science, prostitutes science.
ICNIRP (and related groups  FCC, ARPANSA, Safetycode6) are directly connected with Telecom, and reason why all curiosity is hibernated, by Telecom, and by their product, the entertainment offered by cellphones.

In the article above is mentioned that "we" like the cellphone.
That sentence also includes that those who own a cellphone, generally do not take their accountability for what they create, what they destroy, in nature, in public health, and last but not least: their own health.
Like cigarettes: those who smoke did not take their responsibility and do not, yet, in the present, while they know what the consequences are.
Cellphone users are worse than cigarette smokers. 
They cannot say, ever: We Did Not Know.
GP's neither.
